samples/nrf5340/multiprotocol_rpmsg/README.rst

```diff
@@ -19,6 +19,10 @@ See the following configuration options for more information:
 You might need to adjust the Kconfig configuration of this sample to make it compatible with the peer application.
 For example, :option:`CONFIG_BT_MAX_CONN` must be equal to the maximum number of connections supported by the peer application.
 
+The sample is configured to be energy efficient by disabling:
+ * Serial Console and Logger in :file:`prj.conf`
+ * Unused peripherals in :file:`boards/nrf5340dk_nrf5340_cpunet.overlay`
```

Comment:
```suggestion
The following components have been disabled to make this sample energy-efficient:

* Serial console (in :file:`prj.conf`)
* Logger (in :file:`prj.conf`)
* Unused peripherals (in :file:`boards/nrf5340dk_nrf5340_cpunet.overlay`)
```

Reply:
The case is that Logger is not explicitly disabled in the `prj.conf`, it is just not enabled by anything.
I would like to avoid a situation in which a customer would look in the `prj.conf` searching for specific line disabling logging.

Reply:
I thought this is what CONFIG_LOG=n in there does?

Reply:
It's not there https://github.com/nrfconnect/sdk-zephyr/blob/2ccd9a9a5218a4c10ff61528d585e31f4f2cbeb7/samples/boards/nrf/ieee802154/802154_rpmsg/prj.conf

Reply:
Done. The last line was removed as the `nrf5340dk_nrf5340_cpunet.overlay` is no longer included in this PR.